workspace/libs/shared/ui/src/lib/components/dialogs/confirm-dialog/confirm-dialog.component.ts

```diff
@@ -100,10 +100,12 @@ export class ConfirmDialogComponent implements OnInit, AfterViewInit {
   }
 
   public ngAfterViewInit(): void {
-    const adjustSubmitFocus = document.querySelector(
-      '#submit',
-    ) as HTMLButtonElement;
-    if (adjustSubmitFocus) adjustSubmitFocus.focus();
+    setTimeout(() => {
+      const adjustSubmitFocus = document.querySelector(
+        '#submit',
+      ) as HTMLButtonElement;
+      if (adjustSubmitFocus) adjustSubmitFocus.focus();
+    }, 0);
```

Comment:
Can we define the timeout value in constants.
So that the components will not have the magic values.

Reply:
Remove the `0` it is not needed. The idea is to defer this code until the change detection cycle is complete. Could also be achieved by invoking the `ChangeDetectorRef` to mark the component for change detection, but simply deferring it works.